src/test/java/gov/acwi/wqp/etl/EtlBiodataIT.java

```diff
@@ -57,22 +57,12 @@
 			table=EXPECTED_DATABASE_TABLE_CHECK_TABLE,
 			query=BASE_EXPECTED_DATABASE_QUERY_CHECK_TABLE + "'result_swap_biodata'")
 
-	// <dataType>_swap_biodata tables
-	@ExpectedDatabase(
-			value="classpath:/testResult/wqp/station/station_swap_biodata.xml",
-			assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
-	@ExpectedDatabase( 
-			value="classpath:/testResult/wqp/orgData/orgData.xml", 
-			assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
-	@ExpectedDatabase(
-			value="classpath:/testResult/wqp/activity/activity.xml",
-			assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
-	@ExpectedDatabase(
-			value="classpath:/testResult/wqp/projectData/project_data_swap_biodata.xml",
-			assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
-	@ExpectedDatabase(
-			value="classpath:/testResult/wqp/result/result_swap_biodata.xml",
-			assertionMode= DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	// Biodata base data
+	@ExpectedDatabase(value="classpath:/testResult/wqp/station/station_swap_biodata.xml", assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	@ExpectedDatabase(value="classpath:/testResult/wqp/orgData/orgData.xml", assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	@ExpectedDatabase(value="classpath:/testResult/wqp/activity/activity.xml", assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	@ExpectedDatabase(value="classpath:/testResult/wqp/projectData/project_data_swap_biodata.xml", assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	@ExpectedDatabase(value="classpath:/testResult/wqp/result/result_swap_biodata.xml", assertionMode= DatabaseAssertionMode.NON_STRICT_UNORDERED)
```

Comment:
You need to add the test data for the other afterLoad steps: analyzed, primary keys, and foreign keys.

Reply:
Ah! I would like to see those too - thanks for pointing this out.

Reply:
After verbal follow-up, we'll pass on these tests in the end-to-end IT until we get the finalize step in place on this project.